lib/actions/addDelivery.js

```diff
@@ -37,26 +18,29 @@ function action(msg, cfg) {
         .done();
 
     function addDelivery(response) {
-        var orderBody = response.body;
-        var version = orderBody.version;
-
-        var orderId = msgBody.orderId;
-        var lineItemId = msgBody.lineItemId;
-        var quantity = msgBody.quantity;
-        var parcelId = msgBody.parcelId;
-        var trackingId = msgBody.trackingId;
-
-        if (!orderId) {
+        if (!response.body.count) {
+            throw new Error(util.format("No order with orderNumber %s found", msgBody.orderNumber));
+        }
+        if (!response.body.results[0].id) {
```

Comment:
https://github.com/elasticio/sphereio-api/pull/75#discussion-diff-33228217R21 does this check